src/murmur/Server.cpp

```diff
@@ -1464,17 +1468,22 @@ void Server::message(unsigned int uiType, const QByteArray &qbaMsg, ServerUser *
 
 		MessageHandler::UDPMessageType msgType = static_cast<MessageHandler::UDPMessageType>((buffer[0] >> 5) & 0x7);
 
-		switch (msgType) {
-			case MessageHandler::UDPVoiceCELTAlpha:
-			case MessageHandler::UDPVoiceCELTBeta:
-			case MessageHandler::UDPVoiceSpeex:
-				if (bOpus)
-					break;
-			case MessageHandler::UDPVoiceOpus:
-				processMsg(u, buffer, l);
-				break;
-			default:
-				break;
+		if (msgType == MessageHandler::UDPVoiceSpeex ||
+			msgType == MessageHandler::UDPVoiceCELTAlpha ||
```

Comment:
Please use spaces for alignment, and align msgType with the first line:

    if (msgType == MessageHandler::UDPVoiceSpeex ||
        msgType == MessageHandler::UDPVoiceCELTAlpha ||
        [...]